src/antsibull_core/config.py

```diff
@@ -29,7 +33,7 @@ class ConfigError(Exception):
     pass
 
 
-def find_config_files(conf_files: Iterable[str]) -> list[str]:
+def find_config_files(conf_files: Iterable[StrPath] | Iterable[str]) -> list[str]:
```

Comment:
Does

```suggestion
def find_config_files(conf_files: Iterable[StrPath]) -> list[str]:
```

not work?

Reply:
I tried that first, and for some reason it didn't and I had to use the above. In any case, now it works, don't ask me why :shrug: 2c29d5b2af8762ab81d2156bddf084a94172b6c4